src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -250,6 +251,19 @@ const formConfig = {
             } explanation of benefits`,
           ...applicantInsuranceEOBSchema(true),
         },
+        primaryScheduleOfBenefits: {
+          path: 'insurance-sob',
+          depends: formData =>
+            get('applicantHasPrimary', formData) &&
+            get('applicantPrimaryEOB', formData) === 'noEob',
+          title: formData =>
+            `${nameWording(formData)} ${
+              formData.applicantPrimaryProvider
+            } schedule of benefits`,
+          CustomPage: FileFieldWrapped,
+          CustomPageReview: null,
+          ...applicantInsuranceSOBSchema(true),
+        },
```

Comment:
Adds upload to primary insurance screen and configures schema to use primary language.